aspnetcore/security/authentication/customize-identity-model.md

```diff
@@ -342,15 +342,37 @@ Update *Pages/Shared/_LoginPartial.cshtml* and replace `IdentityUser` with `Appl
 @inject UserManager<ApplicationUser> UserManager
 ```
 
-Update *Areas/Identity/IdentityHostingStartup.cs*  or `Startup.ConfigureServices` and replace `IdentityUser` with `ApplicationUser`.
+Update *Areas/Identity/IdentityHostingStartup.cs* or `Startup.ConfigureServices` and replace `IdentityUser` with `ApplicationUser`.
+
+::: moniker range=">= aspnetcore-2.1 <= aspnetcore-5.0"
```

Comment:
```suggestion
```

Remove moniker unless this doesn't work for 6. If 6 is different, you'd need a moniker for 6

Reply:
resolved